lib/SIL/Verifier/SILVerifier.cpp

```diff
@@ -2800,7 +2800,9 @@ class SILVerifier : public SILVerifierBase<SILVerifier> {
     if (auto *AEBI = dyn_cast<AllocExistentialBoxInst>(PEBI->getOperand())) {
       // The lowered type must be the properly-abstracted form of the AST type.
       SILType exType = AEBI->getExistentialType();
-      auto archetype = OpenedArchetypeType::get(exType.getASTType());
+      auto *DC = F.getDeclContext() ? F.getDeclContext()
+                                    : F.getModule().getSwiftModule();
```

Comment:
This needs to be able to pass the SILFunction's generic signature to work with thunks, etc.